README.md

```diff
@@ -46,10 +46,27 @@ unleash.on('synchronized', () => {
   const variant = unleash.getVariant('app.ToggleY');
 });
 ```
+Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this method multiple times the global instance will be changed. 
 
-Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this
-method multiple times the global instance will be changed. If you prefer to handle the instance
-yourself you should [construct your own Unleash instance](#alternative-usage).
+#### <u>Constructing the Unleash client directly</u>
```

Comment:
Is the `u` tag intended to underline the heading? It doesn't seem to work on GitHub (at least not in the preview), and I think we should leave the header styling up to the markdown renderer anyway 💁🏼

```suggestion
#### Constructing the Unleash client directly
```